harbor-cluster/templates/openshift-resource-locker-ns.yaml

```diff
@@ -0,0 +1,57 @@
+ {{- if .Values.openshift.patchNs.enabled }}
+apiVersion: redhatcop.redhat.io/v1alpha1
```

Comment:
Instead of a namespace patch I suggest using a specific SCC that will be used by Harbor in this specific namespace.

Reply:
I did implement using nonroot scc and give the default sa this scc: https://github.com/itmwiw/harbor/blob/main/harbor-cluster/templates/scc-nonroot-rolebinding.yaml
Is this what you mean or is it something else?

Reply:
the objective is to keep the whole namespace in restricted SCC and to have only harbor use nonroot SCC. so does it work if you keep the rolebinding and remove namespace patch ?

Reply:
also, the rolebinding should be tied to Harbor's service account instead of default service account (if Harbor uses a specific service account of course).

Reply:
yes it works. you can either patch the namespace or use the nonroot scc. One of them is enough. By default, Harbor uses the default sa but it is possible to change it for each componment I think. We can add a global serviceAccount parameter that will be used by all the componments and by the rolebinding. What do you think?

Reply:
yes ok to add a specific serviceaccount, but only for harbor.

Reply:
done